wen-restart/src/wen_restart.rs

```diff
@@ -1005,6 +867,15 @@ pub fn wait_for_wen_restart(config: WenRestartConfig) -> Result<()> {
                     --no-snapshot-fetch",
                     slot, hash, shred_version,
                 );
+                if config.cluster_info.id() == config.wen_restart_coordinator {
+                    aggregate_restart_heaviest_fork(
```

Comment:
why do we need to aggregate on completion here?

Reply:
I think this was your suggestion: make the coordinator aggregate heaviest fork at the end, so we know the status of the cluster from watching the coordinator